src/App.js

```diff
@@ -3,11 +3,13 @@ import { Container } from "react-bootstrap";
 import "./App.css";
 import Footer from "./components/Footer/Footer";
 import "bootstrap/dist/css/bootstrap.min.css";
+import OurTeam from "./components/OurTeam/OurTeam";
 
 function App() {
   return (
     <>
       <Container>
+        <OurTeam />
```

Comment:
Isn't OurTeam component a part of AboutPage container?
![image](https://user-images.githubusercontent.com/81304552/130050029-9de29cc4-7356-4789-88cb-f2b0699f50f6.png)

Reply:
Yes, it is! When Halit accept PR of About, i'm going to `git merge dev` on ourTeam branch then i'll add ourTeam component.